1.regression_modelling/scripts/1.train_models.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Train ElasticNet model to predict each A647 and GOLD feature using the nuclear features
+# 
+# In this notebook, we split the features into each group for nuclear speckle or nucleus and then train a model per nuclear speckle feature using the nuclear features to predict it.
+# 
+# We are looking to find the best nuclear speckle feature that can be predicted using nucleus features.
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+import numpy as np
+import joblib
+from sklearn.linear_model import ElasticNet
+from sklearn.model_selection import KFold, RandomizedSearchCV
+import warnings
+import sys
+import os
+# Ignore the ConvergenceWarnings (only thing that will work 🙃)
+if not sys.warnoptions:
+    warnings.simplefilter("ignore")
+    os.environ["PYTHONWARNINGS"] = "ignore"
```

Comment:
Looks good!

Reply:
Took me forever to get that to work haha, so thank you!